### DISCUSSION TITLE
c

### DISCUSSION THREAD

**Discussion by testcurioues** [Schweinepriester/github-profile-achievements#583] [Q&A] [answered]
Como posso iniciar um projeto em React?

**Comment by LucasLomiento** [comment, marked as answer]
Para iniciar um projeto em React, você precisará ter instalado o Node.js e o npm (gerenciador de pacotes do Node) em sua máquina.

Uma das maneiras mais fáceis de iniciar um projeto React é usando o Create React App, que é um conjunto de ferramentas criadas pelo Facebook para ajudar a configurar um novo projeto React rapidamente.

Para usar o Create React App, basta abrir o terminal e digitar o seguinte comando:

`npx create-react-app nome-do-seu-projeto`
Isso irá criar uma nova pasta com o nome especificado, e dentro dela, uma estrutura básica de arquivos e pastas para um projeto React. Dentro da pasta do seu projeto, execute o comando:


`npm start`
Isso irá iniciar o servidor de desenvolvimento e abrirá seu navegador com a aplicação rodando na rota "http://localhost:3000/".

Agora você pode começar a escrever seu código e a estruturar sua aplicação React.

**Reply by testcurioues** [reply]
wow!
